doc/api/process.md

```diff
@@ -1365,6 +1394,18 @@ system platform on which the Node.js process is running. For instance
 console.log(`This platform is ${process.platform}`);
 ```
 
+## process.postMessage(value[, transferList])
+<!-- YAML
+added: REPLACEME
+-->
+
+This method is available if this is a `Worker` thread, which can be tested
```

Comment:
I'm not sure if all the worker stuff should go on `process` and not on the `Worker` object. `process` is already really big and `process.postMessage` sounds a little confusing.

Reply:
You mean, make all of these statics on the `Worker` constructor? Or the `worker` module? I’m okay with that, it doesn’t really make a difference and I agree that `process` is already pretty big.

Reply:
The worker module and the constructor both sound fine